keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactPin.kt

```diff
@@ -15,7 +14,10 @@ data class PinnedEnvironment(
   val deliveryConfigName: String,
   val targetEnvironment: String,
   val artifact: DeliveryArtifact,
-  val version: String
+  val version: String,
+  val pinnedBy: String?,
+  val pinnedAt: Long?,
```

Comment:
This should be an `Instant?`, no?

Reply:
I have no idea whether this code would actually impact what gets serialized to clients but I would strongly prefer the API continue to only return ISO timestamps

Reply:
Ah yes, I'll fix. Good reminder. I removed my `// todo eb: type? fix?` comment too soon 😆